Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ButtonData.xml

```diff
@@ -7082,4 +7082,125 @@
         <EditorCategories value="Race:Protoss"/>
         <HotkeyAlias value="AP_ArchonSiphon"/>
     </CButton>
+    <CButton id="AP_ScoutNerazimCloak">
+        <Icon value="AP\Assets\Custom\Textures\btn-ability-terran-cloak-gold.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="CloakOff"/>
+        <HotkeySet value="Cloak"/>
+    </CButton>
+    <CButton id="AP_ScoutNerazimCloakUpgrade">
+        <Icon value="AP\Assets\Custom\Textures\btn-ability-terran-cloak-gold.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="CloakOff"/>
+        <HotkeySet value="Cloak"/>
+    </CButton>

```

Comment:
You're using HotkeyAlias for decloaking there (and shared with actual decloak on the same unit)